src/pages/public/RegistrationForms/MembershipForm/index.js

```diff
@@ -208,7 +208,7 @@ const MembershipFormContainer = (props) => {
       fname: first_name,
       lname: last_name,
       major: memberType === "UBC" || memberType === "UNI" ? major : "",
-      email: email.toLowerCase(),
```

Comment:
@ddennis924 

why was this change introduced? I don't know if you're still working on the email stuff but if you aren't let's set up a meeting with you Victor and I to transfer knowledge.